third-party/realdds/src/dds-device-impl.cpp

```diff
@@ -429,7 +429,8 @@ struct dds_device::impl::init_context
 bool dds_device::impl::init()
 {
     // We expect to receive all of the sensors data under a timeout
-    rsutils::time::timer timer( std::chrono::seconds( 5 ) );
+    rsutils::time::timer timer( std::chrono::seconds(
```

Comment:
Inconsistent seconds / milliseconds.
Separate to two statements for readability.

Reply:
Right. Done.